recipes/icon/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "icon" %}
+{% set version = "9.5.2" %}
+{% set build = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gtownsend/icon/archive/refs/heads/master.zip  # [linux64]
+  sha256: 6c118b2dcc26b6ea3c4129cc82d110a275d3c806749211773fe34c106949efb0  # [linux64]
+
+build:
+  number: {{ build }}
```

Please keep this as a plain number 
```suggestion
  number: 0
```